src/ansys/geometry/core/designer/body.py

```diff
@@ -1611,13 +1614,72 @@ def plot(  # noqa: D102
         pl.show(screenshot=screenshot, **plotting_options)
 
     def intersect(self, other: Union["Body", Iterable["Body"]], keep_other: bool = False) -> None:  # noqa: D102
-        self.__generic_boolean_op(other, keep_other, "intersect", "bodies do not intersect")
+        # self.__generic_boolean_op(other, keep_other, "intersect", "bodies do not intersect")
+        self.__generic_boolean_command(
+            other, keep_other, False, "intersect", "bodies do not intersect"
+        )
```

Comment:
The fallback to the old implementation is still need when interacting with 25R1 and earlier versions. We need to handle this through a try except based on the result of the ``@min_backend_version`` decorator.